apps/client/src/app/pages/list-details/list-details/list-details.component.html

```diff
@@ -184,14 +184,24 @@ <h2>{{list.name}}</h2>
                     nz-tooltip nzSize="small">
               <i nz-icon nzType="share-alt"></i>
             </button>
-            <button (cbOnSuccess)="afterListTextCopied()" *ngIf="display$ | async as listDisplay"
-                    [cbContent]="getListTextExport(listDisplay, list)" [nzTitle]="'LIST.Copy_list_as_text' | translate"
-                    ngxClipboard
+            <button (click)="$event.stopPropagation();copyTextExport(listDisplay, list)"
+                    *ngIf="display$ | async as listDisplay"
+                    [nzTitle]="'LIST.Copy_list_as_text' | translate"
                     nz-button nz-tooltip
                     nzShape="circle"
                     nzSize="small">
               <i nz-icon nzType="copy"></i>
             </button>
+            <!-- cannot have async pipes in a template action https://stackoverflow.com/questions/40300115/how-to-pass-async-variable-in-template-action-function -->
+            <input  #servername [value]="server$ | async" type="hidden" />
+            <button (click)="$event.stopPropagation();copyJSONExport(servername.value, listDisplay, list)"
+                    *ngIf="display$ | async as listDisplay"
+                    [nzTitle]="'LIST.Copy_list_as_json' | translate"
+                    nz-button nz-tooltip
+                    nzShape="circle"
+                    nzSize="small">
+              <i nz-icon nzType="snippets"></i>
+            </button>
```

Comment:
You can simply wrap this inside a parend container component, having hidden components is too hacky for me to accept it ;)

```suggestion
          <ng-container *ngIf="server$ | async as server">
            <button (click)="$event.stopPropagation();copyJSONExport(server, listDisplay, list)"
                    *ngIf="display$ | async as listDisplay"
                    [nzTitle]="'LIST.Copy_list_as_json' | translate"
                    nz-button nz-tooltip
                    nzShape="circle"
                    nzSize="small">
              <i nz-icon nzType="snippets"></i>
            </button>
          </ng-container>
```